Configurations/unix-Makefile.tmpl

```diff
@@ -710,7 +710,8 @@ generate: generate_apps generate_crypto_bn generate_crypto_objects \
 
 doc-nits:
 	(cd $(SRCDIR); $(PERL) util/find-doc-nits -n -p ) >doc-nits
-	if [ -s doc-nits ] ; then cat doc-nits; rm doc-nits ; exit 1; fi
+	@if [ -s doc-nits ] ; then cat doc-nits; rm doc-nits ; exit 1; \
+	else echo 'find-doc-nits: no errors.' ; fi
```

Comment:
Is it intentional that `doc-nits` is removed only in case of errors?
Should this target be `.PHONY`?

Ask because this happened to me while improving the documentation on #6514 : 
- initial commit opening the PR
- `make  doc-nits`: no errors (well before #6517 , literally no error message), `doc-nits` is left there empty
- further changes to pod files to address feedback on the PR
- `make doc-nits`:
~~~
make: 'doc-nits' is up to date.
~~~
- manually remove the empty `doc-nits` and rerun `make doc-nits`

Reply:
@mspncp in the PR description you have the "Already done" usecase, but the problem is that given that the target has no dependency, it will always say "is up to date" even if there were changes in the .pod files.

Reply:
@richsalz old the ready label please!

Reply:
Right, output to `doc-nits.log` instead and only remove it on success.

Also, I would prefer if the success message had `doc-nits` rather than `find-doc-nits`

Reply:
@romen thanks for your suggestion. After a while one gets so used to removing the doc-nits file that one forgets to think of a better way to do it. I will rething my pr. 

> @richsalz hold the ready label please!

And don't worry. By convention, committers normally merge their pull requests themselves, so it's up to me to merge.

@levitte no problem, I will change it.

Reply:
> Right, output to doc-nits.log instead and only remove it on success

Actually, I would prefer if `find-doc-nits` would return an error code. Then it would not be necessary to redirect the output and check the size of the output file in the first place. But I'm not sure whether ~~the force~~ the perl-fu is already strong enough inside me to change it. (I have to confess that I did a perl tutorial recently ;-) )